packages/botonic-core/index.d.ts

```diff
@@ -0,0 +1,6 @@
+export interface Route {
+  path: string;
+  payload?: RegExp| string;
+  text?: string;
+  action: React.ReactNode;
+}
```

Comment:
If this component is representing the 'router' of Botonic, we need ti add intent variable, I don't remember right now but we had the option of passing an attribute called 'type', where we defined the type of the input, like: 'type: image', or 'type: text'. I don't know if it's necessary to declare it, but to keep in mind.

Reply:
right! thanks
I'm adding this in future PR
```
type StringMatcher = RegExp| string | ((data: string) => boolean);

export interface Route {
  path: StringMatcher;
  payload?: StringMatcher;
  text?: StringMatcher;
  type?: StringMatcher;
  intent?: StringMatcher;
  session?: (session: any) => boolean;
  action: React.ReactNode;
}
```